.github/issue_template.md

```diff
@@ -0,0 +1,11 @@
+## Expected Behavior
```

Comment:
Not a bad idea, how about this? -> _"If you're requesting a new feature, tell us why you'd like this feature."_